autode/conformers/conformers.py

```diff
@@ -8,6 +8,9 @@
 from autode.mol_graphs import make_graph, is_isomorphic
 from autode.geom import calc_heavy_atom_rmsd
 from autode.log import logger
+from autode.units import ha
+from math import exp
+from scipy.constants import Boltzmann
```

Comment:
```suggestion
```
No need to import `ha` (see below), I would use `np.exp` for consistency and I'd prefer to just define k_b in the function.
it's not obvious what units `scipy.constants.Boltzmann` is in..